### ISSUE TITLE
ChromeDriverManager not found.

### ISSUE THREAD

**Issue by Erik172** [SergeyPirogov/webdriver_manager#535]
I created a scraper with selenium a few weeks ago and `webdriver-manager` works fine until these days that comes out an error that I could not solve, the only solution that I found is to remove `webdriver-manager`.

**Sytem**

```
Windows 10
Python 3.11.4
```

**Versions:**
```
webdriver-manager==3.8.6
selenium==4.10.0
```

**Code:**
```
def __init__(self):
        '''
        This function is used to initialize the webdriver and the options for the webdriver
        '''
        chrome_options = Options()
        chrome_options.add_argument('--headless')
        chrome_options.add_argument('--window-size=1920x1080')
        chrome_options.add_argument(f'user-agent={UserAgent().random}')

        self.driver = webdriver.Chrome(ChromeDriverManager().install(), options=chrome_options)
```

**Error:**

```
2023-07-18 14:35:58 [WDM] INFO: ====== WebDriver manager ======
Unhandled error in Deferred:
2023-07-18 14:35:59 [twisted] CRITICAL: Unhandled error in Deferred:

Traceback (most recent call last):
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\scrapy\crawler.py", line 240, in crawl
    return self._crawl(crawler, *args, **kwargs)
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\scrapy\crawler.py", line 244, in _crawl
    d = crawler.crawl(*args, **kwargs)
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\twisted\internet\defer.py", line 1947, in unwindGenerator
    return _cancellableInlineCallbacks(gen)
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\twisted\internet\defer.py", line 1857, in _cancellableInlineCallbacks
    _inlineCallbacks(None, gen, status, _copy_context())
--- <exception caught here> ---
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\twisted\internet\defer.py", line 1697, in _inlineCallbacks
    result = context.run(gen.send, result)
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\scrapy\crawler.py", line 128, in crawl
    self.spider = self._create_spider(*args, **kwargs)
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\scrapy\crawler.py", line 140, in _create_spider
    return self.spidercls.from_crawler(self, *args, **kwargs)
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\scrapy\spiders\__init__.py", line 53, in from_crawler
    spider = cls(*args, **kwargs)
  File "C:\Users\erik1\Documents\Projects\bogota-apartments\bogota_apartments\spiders\metrocuadrado_spider.py", line 30, in __init__
    self.driver = webdriver.Chrome(ChromeDriverManager().install(), options=chrome_options)
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\webdriver_manager\chrome.py", line 39, in install
    driver_path = self._get_driver_path(self.driver)
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\webdriver_manager\core\manager.py", line 26, in _get_driver_path
    binary_path = self.driver_cache.find_driver(driver)
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\webdriver_manager\core\driver_cache.py", line 82, in find_driver
    metadata = self.load_metadata_content()
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\webdriver_manager\core\driver_cache.py", line 110, in load_metadata_content
    return json.load(outfile)
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 293, in load
    return loads(fp.read(),
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

2023-07-18 14:35:59 [twisted] CRITICAL:
Traceback (most recent call last):
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\twisted\internet\defer.py", line 1697, in _inlineCallbacks
    result = context.run(gen.send, result)
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\scrapy\crawler.py", line 128, in crawl
    self.spider = self._create_spider(*args, **kwargs)
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\scrapy\crawler.py", line 140, in _create_spider
    return self.spidercls.from_crawler(self, *args, **kwargs)
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\scrapy\spiders\__init__.py", line 53, in from_crawler
    spider = cls(*args, **kwargs)
  File "C:\Users\erik1\Documents\Projects\bogota-apartments\bogota_apartments\spiders\metrocuadrado_spider.py", line 30, in __init__
    self.driver = webdriver.Chrome(ChromeDriverManager().install(), options=chrome_options)
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\webdriver_manager\chrome.py", line 39, in install
    driver_path = self._get_driver_path(self.driver)
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\webdriver_manager\core\manager.py", line 26, in _get_driver_path
    binary_path = self.driver_cache.find_driver(driver)
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\webdriver_manager\core\driver_cache.py", line 82, in find_driver
    metadata = self.load_metadata_content()
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\site-packages\webdriver_manager\core\driver_cache.py", line 110, in load_metadata_content
    return json.load(outfile)
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 293, in load
    return loads(fp.read(),
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\erik1\AppData\Local\Programs\Python\Python311\Lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**if add try:**
```
        try:
            self.driver = webdriver.Chrome(ChromeDriverManager().install(), options=chrome_options)
        except Exception as e:
            print(f'Error: {e}')
            os._exit(1)
```
```
2023-07-18 14:37:45 [WDM] INFO: ====== WebDriver manager ======
Error: Expecting value: line 1 column 1 (char 0)
```

the only way it works is by removing `ChromeDriverManager().install()`

**Comment by david-engelmann** [comment]
I believe this relates to this [open PR](https://github.com/SergeyPirogov/webdriver_manager/pull/537)

**Comment by SergeyPirogov** [comment]
Fixed in 3.9.0
